.all-contributorsrc

```diff
@@ -0,0 +1,24 @@
+{
+  "files": [
+    "README.md"
+  ],
+  "imageSize": 100,
+  "commit": false,
+  "contributors": [
+    {
+      "login": "eleith",
+      "name": "eleith",
+      "avatar_url": "https://avatars.githubusercontent.com/u/284832?v=4",
+      "profile": "http://eleith.com",
+      "contributions": [
+        "infra"
+      ]
+    }
+  ],
+  "contributorsPerLine": 7,
```

Comment:
```suggestion
  "contributorsPerLine": 7,
  "badgeTemplate": ""
```

Can this disable the badge generation? See https://allcontributors.org/docs/en/bot/configuration

Reply:
I will apply the suggestions and remove the badge below manually too if necessary.